### PR TITLE
Feature/mcs 2015 send documents in task

### PR DESCRIPTION
**Pull request by mtrofimiuk**

### CODE REVIEW

File: documents/tasks/mails.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        order = Order.objects.get(id=order_id)
+        documents_to_send = [
+            document for document in documents if email in document.receivers_emails
+        ]
+
+        attachments_as_blobs = [
+            self.bucket.blob(document.blob_name) for document in documents_to_send
+        ]
+
+        send_log = DocumentSendLog.objects.select_for_update().filter(
```
Comment (eshlox) [inline documents/tasks/mails.py]: Why do we need `select_for_update` here?

File: documents/tasks/mails.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            order_id=order_id, email=email
+        )
+
+        if not DocumentSendLog.documents_sent(order_id=order_id, email=email):
```
Comment (eshlox) [inline documents/tasks/mails.py]: Is it possible to get this value from `send_log`? 🤔

File: documents/tasks/mails.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            send_mail(
```
Comment (eshlox) [inline documents/tasks/mails.py:32]: What happens if Mailgun returns an error here?

Comment (mtrofimiuk) [reply documents/tasks/mails.py:32]: The transaction will be rolled back and send log will still have `sent=False`

**Pull request merged**